src/EncodeHQ-CBR/EncodeHQ-CBR.cpp

```diff
@@ -285,7 +285,7 @@ try { //Giant try block around all code to get error messages
   Array2D uDiff(format.chromaShape()); // used for PSNR calculation only
   Array2D vDiff(format.chromaShape()); // used for PSNR calculation only
 
-  int frame = 0;
+  unsigned long long frame = 0;
```

Comment:
At some point if you intend to maintain this codebase I would suggest including `stdint` and using sized ints in place of these. Technically the size of types like `unsigned long long int` are archetecture dependent.

Reply:
Doesn't need to be now, though

Reply:
Okay, that's good to know, I'll open an issue so this doesn't get forgotten :)